api/local.ts

```diff
@@ -1,16 +1,15 @@
 /**
  * Common to both server.ts and local.ts files
  */
-import * as propertiesVolume from '@hmcts/properties-volume'
-import * as config from 'config'
 import { app } from './application'
-import {getConfigValue, getIDamSecret, initialiseSecrets} from './configuration'
+import {getConfigValue, initialiseSecrets} from './configuration'
```

Comment:
don't need the `initialiseSecrets` function anymore

Reply:
Done